lib/ssh_data/public_key.rb

```diff
@@ -20,6 +20,14 @@ module PublicKey
       ALGO_ED25519, ALGO_SKECDSA256, ALGO_SKED25519
     ]
 
+    DEFAULT_SK_VERIFY_OPTS = {
```

Comment:
Might be worth a comment on why these are the defaults.

Reply:
Also..you tell me..are these constants best placed in here or in the module you added for security-key specific bits? I guess it is a question of whether these constants/etc are ever really needed outside of security-key specific internal code.

Reply:
That's a good idea, that module didn't exist till after I shoved those in `PublicKey` and never thought to move them to a better place.